Gemfile

```diff
@@ -1,10 +1,34 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-group :development, :test do
-  gem 'bodeco_module_helper', :git => 'https://github.com/bodeco/bodeco_module_helper.git'
-  gem 'yard'
-  gem 'faraday'
-  gem 'faraday_middleware'
+group :test do
+  gem 'rake'
+  gem 'puppet', ENV['PUPPET_VERSION'] || '~> 3.8.0'
+  gem 'rspec-puppet', git: 'https://github.com/rodjek/rspec-puppet.git'
+  gem 'rspec-puppet-augeas'
+  gem 'ruby-augeas'
```

Comment:
The augeas gem is part of the default gemset that modulesync gives people. @igalic should we have a boolean to toggle this?